.docker/testdb.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/bash
 set -e
 
-psql "postgresql://$POSTGRES_USER@:5432/$POSTGRES_DB" -v ON_ERROR_STOP=1 <<-EOSQL
```

Comment:
Why is `$POSTGRES_DB` part of URI no longer needed?

Reply:
We are creating DB, users and granting privileges. Hence, we don't need to specify any database to connect to.